Umbraco-Cloud/Set-Up/Project-settings/index.md

```diff
@@ -28,7 +28,7 @@ Binding hostnames to your Umbraco Cloud project is done from the Umbraco Cloud P
 
 ## [Certificates](../Manage-Hostnames/Security-Certificates)
 
-All hostnames added to your Umbraco Cloud environments will automatically be assigned a security certificate - we call this feature [Umbraco Latch](../Umbraco-Latch).
+All hostnames added to your Umbraco Cloud environments will have automatic TLS (HTTPS) certificates, meaning that the certificates issued by Cloudflare will be automatically renewed annually as part of the Umbraco Cloud service.
 
 Sometimes you might want to upload and bind your own security certificate to your hostnames. This can be  done from the **Certificates** page.
```

Comment:
The link here is actually to the article on working with custom certificates. Updated the text accordingly - will most likely update the name of the article at a later point as well.

```suggestion
Sometimes you might want to upload and bind your own custom security certificate to your hostnames instead of using the TLS (HTTPS) certificates provided by the Umbraco Cloud service. This can be done from the **Certificates** page.
```